step.yml

```diff
@@ -82,12 +82,19 @@ inputs:
 
         __UNCHECK EVERY SCOPE BOX__ when creating this token. There is no reason this token needs access to private information.
       is_sensitive: true
+  - xcconfig:
+    opts:
+      title: Custom xcconfig file to add to Carthage environment
+      description: |-
+        Use this input to provide an xcconfig file as a workaround for Xcode 12 issue. See [the Github issue](https://github.com/Carthage/Carthage/issues/3019) for more information.
```

Comment:
Use this input to provide an `xcconfig` file as a workaround for the Xcode 12 issue. For more information, see [the Github issue](https://github.com/Carthage/Carthage/issues/3019).